core/src/main/scala/mongo4cats/collection/queries/WatchQueryBuilder.scala

```diff
@@ -120,6 +122,17 @@ object WatchQueryBuilder {
     def stream =
       boundedStreamF(1).translate(transform)
 
+    def updateStreamAttempt[A: BsonDecoder](onError: BsonDecodeError => G[Unit]) =
+      boundedStreamF(1)
```

Comment:
You couldn't get a handler for these errors https://github.com/precog/support-incidents/issues/202#issuecomment-1217689209 to work?

Reply:
I haven't attempted it. Planning to address this in [#1844](https://github.com/precog/services/issues/1844)